src/utils/useDebouncedEffect.ts

```diff
@@ -1,7 +1,17 @@
 // credits to https://stackoverflow.com/questions/54666401/how-to-use-throttle-or-debounce-with-react-hook
 import { useCallback, useEffect } from 'react';
 
-export const useDebouncedEffect = (effect, delay, deps) => {
+/**
+ * Produces a debounced version of `useEffect`
+ * @param effect - callback function to run within the `useEffect`
+ * @param delay - time delay in ms
+ * @param deps - dependency array
+ */
```

Comment:
Example: src/utils/countryCurrency/getCurrencySymbolByCode.ts >> `getCurrencySymbolByCode()`

It could be useful to explain what the parameters to this function mean. `langCode` and `currency` are sulf explanatory, but what does `number` mean?